lib/glific/third_party/gigalixir.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Glific.Gigalixir do
+  @moduledoc """
+  This Module contains functions that communicate with Gigalixir via Gigalixir API
+  """
+
+  @endpoint "https://api.gigalixir.com/api/apps/glific-staging/domains"
```

Comment:
We have to move the endpoint also to a environment variable just like username and api_key, because app name can change. In staging the endpoint will be `apps/glific-staging` but in production it is `apps/glific`